<launch>


  <arg name="rviz" default="true" />
  <!-- <rosparam command="load" file="$(find sglc)/config/config_pgo.yaml" /> -->
  <group if="$(arg rviz)">
    <node launch-prefix="nice" pkg="rviz" type="rviz" name="rviz" args="-d $(find sglc)/rviz_cfg/pgo.rviz" />
  </group>

  <node 
		pkg="sglc" 
		type="demo_pgo_loop_correction" 
		name="demo_pgo_loop_correction" 
		output="screen" 
	/>
  <param name="lidar_path" type="string" value = "/home/wangneng/DataFast/kitti/sequences/00/velodyne/"/>  
  <param name="label_path" type="string" value = "/home/wangneng/SLAM/loop_closure/SegNet4D_predicitions/kitti/sequences/00/predictions/"/>
  <param name="pose_path" type="string" value = "$(find sglc)/poses/kitti00_kissicp.txt"/>
  <param name="config_path" type="string" value = "$(find sglc)/config/config_kitti_graph.yaml"/>
  <param name="map_voxel_size" type="double" value = "1"/>
  <param name="file_name_length" type="int" value = "6"/>
  <param name="sub_frame_num" type="int" value = "5"/>


</launch>